README.md

```diff
@@ -52,9 +55,23 @@ cd django-DefectDojo
 ./dc-up.sh postgres-redis
 # obtain admin credentials. the initializer can take up to 3 minutes to run
 # use docker-compose logs -f initializer to track progress
-docker-compose logs initializer | grep "Admin password:"
+docker compose logs initializer | grep "Admin password:"
+```
+## For older verion Compose V1
```

Comment:
```suggestion
## For older version Compose V1
```

Reply:
Opps! The typo!! 

@rndmh3ro Thanks for your peer review. :D